lib/README.md

```diff
@@ -9,7 +9,7 @@ class is the responsable by implements your node behavior. The following methods
 __must be__ implemented:
   - getNodeRepresentationPath
   - getMetadata
-  - getLocaleData
+  - getLocalesPath
```

Comment:
Would be nice to explain on somewhere in the README how the internationalization works, I mean, what the node developer should do.